src/windows/export.py

```diff
@@ -199,14 +199,19 @@ def __init__(self):
         self.profile_paths = {}
         for profile_folder in [info.USER_PROFILES_PATH, info.PROFILES_PATH]:
             for file in os.listdir(profile_folder):
-                # Load Profile
                 profile_path = os.path.join(profile_folder, file)
-                profile = openshot.Profile(profile_path)
+                try:
+                    # Load Profile
+                    profile = openshot.Profile(profile_path)
```

Comment:
For this change, what sort of invalid profile file does `openshot.Profile()` throw a `RuntimeError` on?

Because, in my testing, it does one of two things, only.

1. **Completely ignores** lines with unrecognized fields, e.g.
    ```
    biglyness_x=9000
    biglyness_y=8999
    ```
1. **Crashes OpenShot hard** the moment it encounters malformed data, e.g.
    ```
    biglyness_x.9000
    biglyness_y?8999
    ```

It's not what I would term the most _robust_ parser, nor is the format particularly well-defined to begin with, but I couldn't find any evidence that it would ever throw any sort of exception. Be nice if it _did_, rather than crashing, I'm just worried that may be wishful thinking.

Not **necessarily** an argument against the `try`/`except` as a bit of future-proofing, of course...

Reply:
This has been resolved, I think, in that I accept @kartchnb 's later explanation of "this works with the corresponding libopenshot changes" as more than sufficient answer to my question.